src/mixins/reader.js

```diff
@@ -50,18 +49,25 @@ export default {
             if(!article) return null;
             return `chapter-element-${article.chapterId}`;
         },
-        scrollToChapter(chapterId, offset = -100) {
```

Comment:
Works very well on mobile devices but with this value you won't see the chapter number on desktops.

Reply:
I will revert to the old value, and instead make it so the header does not hide when the app scrolls to chapter, but only when the user himself scrolls down

Reply:
The required changes have been pushed